stack.yaml

```diff
@@ -24,19 +22,16 @@ extra-deps:
 - hslua-objectorientation-2.0.1
 - hslua-packaging-2.0.0
 - lua-2.0.1
+- mintty-0.1.2
```

Comment:
Why not 0.1.3, which is on Hackage?

Reply:
mintty is not required on linux, on windows 0.1.3 errors on the CI.

Reply:
This would leave mintty in its current version and some haskell windows dev could take a stab at fixing the build issue and testing on windows.

Reply:
Here is the error: https://github.com/jgm/pandoc/runs/4240666883?check_suite_focus=true#step:4:7326